includes/class-sensei-course.php

```diff
@@ -697,8 +699,59 @@ public function add_column_headings( $defaults ) {
 			$new_columns['date'] = $defaults['date'];
 		}
 
+		// Make sure other sensei columns stay directly behind the new columns.
+		$other_sensei_culumns = [
```

Comment:
There's a typo here`$other_sensei_culumns` -> `$other_sensei_columns`.

Reply:
Thank you, corrected.